modules/jwplayerBidAdapter.js

```diff
@@ -50,4 +68,76 @@ export const spec = {
   // onBidderError: function({ error, bidderRequest }) {}
 };
 
+function buildRequest(bidRequest, bidderRequest) {
+  const {params} = bidRequest;
+
+  const videoAdUnit = deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = deepAccess(bidRequest, 'params.video', {});
```

Comment:
no `params.video`. That's an old way of configuring video bids. We won't support it